### PR TITLE
Added documentation to transposition.rs

### PR DESCRIPTION
**Pull request by scriptandcompile**

### CODE REVIEW

**Review by alexfertel — DISMISSED** [review]

**Comment by alexfertel** [comment]
Happy to merge once the workflow passes!

**Comment by scriptandcompile** [comment]
Screwed that up when I moved the test over =P

File: src/ciphers/transposition.rs [hunk 1]
```diff
@@ -82,11 +79,6 @@ pub fn transposition(key: &str, input: &str) -> String {
 mod test {
     use super::*;
 
-    #[test]
-    fn test_word() {
-        assert_eq!("OMLERX", transposition("key", "lorem"));
-    }
```
Comment (alexfertel) [inline src/ciphers/transposition.rs]: Since this tests a different word, let's keep it. Otherwise, let's add it to the example as well.

**Comment by scriptandcompile** [comment]
I keep forgetting to do a cargo fmt =P

**Review by alexfertel — APPROVED** [review]

**Comment by alexfertel** [comment]
> I keep forgetting to do a cargo fmt 

This should be configurable in your editor of preference, so that you format on save 👀 (some people don't like it, but I strongly recommend it)

**Pull request merged**